src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -100,32 +100,8 @@ private static VersionInfo GetVersionInfo()
             loaderAssemblyName: typeof(OnnxTransform).Assembly.FullName);
         }
 
-        public static IDataTransform Create(IHostEnvironment env, IDataView input, string modelFile, int? gpuDeviceId = null, bool fallbackToCpu = false)
```

Comment:
>public static IDataTransform Create(IHostEnvironment env, IDataView input, string modelFile, int? gpuDeviceId = null, bool fallbackToCpu = false) [](start = 8, length = 145)

This is an "odd" method, so it seems good to remove, so are we instead working via the extension methods and `MLContext`?

Reply:
I'm not sure about extension methods, but in tests I switch to using `OnnxTransform()` constructor with same parameters and then calling `Transform` method on DataView.
I think it's better to get rid of this Create methods, and use either estimator, or since estimator is trivial, just call directly transform consturctor.

---
In reply to: [245457717](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245457717) [](ancestors = 245457717)

Reply:
Yes, probably.

---
In reply to: [245458012](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245458012) [](ancestors = 245458012,245457717)